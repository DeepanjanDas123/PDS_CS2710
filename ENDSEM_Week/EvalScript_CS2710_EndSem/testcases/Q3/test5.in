GenerateTreeEnhanced
33
2 1
f 3
K 4
O 4
Z 4
d 6
u 6
t 12
L 18
p 20
m 21
C 22
D 22
S 23
H 24
z 24
E 25
c 25
v 26
G 29
b 30
n 31
s 35
5 36
R 40
T 42
g 42
X 45
q 45
7 46
B 48
_ 49
o 49
Quit
