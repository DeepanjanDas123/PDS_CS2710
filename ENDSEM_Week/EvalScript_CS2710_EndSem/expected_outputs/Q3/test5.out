((((g,(C,D)),(X,q)),((7,(S,H)),(B,(z,(t,(d,u)))))),(((_,o),((E,c),(v,G))),(((b,n),((((K,O),(Z,(2,f))),L),s)),((5,R),((p,m),T)))));
